packages/valory/connections/ledger/connection.py

```diff
@@ -18,27 +18,6 @@
 #
 # ------------------------------------------------------------------------------
 
-
```

Comment:
Why remove the header?

Reply:
It was duplicated on a previous PR, there is a header already